crates/rolldown_plugin_dynamic_import_vars/src/to_glob.rs

```diff
@@ -6,15 +6,31 @@ use oxc::{
   codegen::CodeGenerator,
   syntax::operator::BinaryOperator,
 };
-use regex::Regex;
 use std::path::Path;
-use std::sync::LazyLock;
 
 // Disallow ./*.ext
-static OWN_DIRECTORY_STAR_REGEX: LazyLock<Regex> = LazyLock::new(|| {
-  let pattern = r"^\./\*\.\w+$";
```

Comment:
Not block this PR. But using `w` would prevent users from using non-ascii file names. Is this intended?

Reply:
it's for file ext, so I thinks it's as expected